pkg/processor/runtime/python/py/test_wrapper.py

```diff
@@ -243,8 +264,13 @@ def _send_event(self, event):
         if not isinstance(event, dict):
             event = self._event_to_dict(event)
 
+        # on python 3.6, msgpack 0.6.1 default `use_bin_type` to False.
+        # on python > 3.6, msgpack 1.0.2 default `use_bin_type` to True.
+        # to be as close to the processor when it comes to sending events, we need `use_bin_type` to be False
+        packer = msgpack.Packer(use_bin_type=False)
```

Comment:
Would love to understand that better.
I got the impression that this should be `True`

Reply:
actually, it would work both ways, will turn into True since it implements msgpack spec 2.0 (newer)